slides/08-Platform-Hub/1020-LetsGetOnboarded.md

```diff
@@ -0,0 +1,8 @@
+## Let's get onboarded!
+
+Please follow these step to onboard yourself to the Platform Hub:
+
+ 1. Go to [access-acp.digital.homeoffice.gov.uk](https://access-acp.digital.homeoffice.gov.uk) to download and run `ACP OP` VPN profile.
```

Comment:
There is no `ACP OP` VPN profile – did you mean `ACP Operations`? If so, will they have access to this?